src/utils/Linkify.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @copyright Copyright (c) 2021 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
+ * @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+import linkifyStr from 'linkify-string'
+
+/**
+ * Linkify text
+ *
+ * @param {String} text The text to linkify
+ * @returns {object} The linkified object
```

Comment:
```suggestion
 * @param {string} text The text to linkify
 * @returns {string} The linkified string
```

The function `linkifyStr()` returns a string (ref https://linkify.js.org/docs/linkify-string.html).